File: android/app/src/main/java/com/sts/fble/Model/NativeComModel.java
package com.sts.fble.Model;

import com.sts.fble.Common.APIConsts;

import org.json.JSONException;
import org.json.JSONObject;

import java.io.Serializable;

public class NativeComModel implements Serializable {
    public String command = "";
    public String passValue = "";

    public  NativeComModel(JSONObject json){
        try {
            command = json.getString(APIConsts.command);
            passValue = json.getString(APIConsts.passValue);
        } catch (JSONException e) {
            e.printStackTrace();
        }
    }
}

File: lib/Model/NativeComModel.dart
import 'package:fble/Common/APIConsts.dart';

class NativeComModel {
  String command = '';
  String passValue = '';

  NativeComModel();

  factory NativeComModel.fromJSON(Map<String, dynamic> res){
    NativeComModel model = new NativeComModel();
    model.command = res[APIConsts.command];
    model.passValue = res[APIConsts.passValue];

    return model;
  }

  Map<String, dynamic> toJSON() => {
    APIConsts.command : command,
    APIConsts.passValue : passValue
  };
}